### PR TITLE
Status for course

### PR DESCRIPTION
**Pull request by MartinBentley**
### Context
This change is to surface the course status.
### Changes proposed in this pull request
The UcasCourse table now has a Status. (other fields have been added but these are not require atm)
The importer now provides the payload with the course status.
### Guidance to review
Status should now be return from the Api as part of the Course data.
The School object will now hold the status as this is need for the UI

### CODE REVIEW

File: src/ManageCourses.Api/Services/Data/DataService.cs [hunk 1]
```diff
@@ -426,7 +421,8 @@ private Course LoadCourse(IReadOnlyList<UcasCourse> courseRecords)
                 Address4 = courseRecord.UcasCampus.Addr4,
                 PostCode = courseRecord.UcasCampus.Postcode,
                 ApplicationsAcceptedFrom = courseRecord.CrseOpenDate,
-                Code = courseRecord.UcasCampus.CampusCode
+                Code = courseRecord.UcasCampus.CampusCode,
+                Status = courseRecord.Status
```
Comment (timabell) [inline src/ManageCourses.Api/Services/Data/DataService.cs]: Add the trailing commas to lists to make diffs cleaner in future.

Minor.

File: tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs [hunk 2]
```diff
@@ -254,6 +254,26 @@ public void GetCoursesShouldReturnData()
...
         }
+        [Test]
+        public void GetCoursesShouldReturnStatus()
+        {
+            const int numOrgs = 5;
+            const int numCourses = 6;
+            LoadData(TestUserEmail1, numOrgs, numCourses);
+            var orgList = DataService.GetOrganisationsForUser(TestUserEmail1).ToList();
+            Assert.IsTrue(orgList.Count == numOrgs);
+            Assert.IsTrue(orgList.All(c => c.TotalCourses == numCourses));
```
Comment (timabell) [inline tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs]: Use fluent assertions throughout for nicer syntax and better test failure messages.

Reject.

File: tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs [hunk 3]
```diff
@@ -435,7 +455,8 @@ private static UcasPayload GetUcasCoursesPayload()
                     new UcasCourse
                     {
                         InstCode = InstCode1,
-                        CrseCode = "CourseCode_1"
+                        CrseCode = "CourseCode_1",
+                        Status = "N"
```
Comment (timabell) [inline tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs]: you can see in this the effect on the diff of not previously having the trailing comma, it turns what would have been a one-line addition into a two line addition and one line deletion

File: src/ManageCourses.Api/Services/Data/DataService.cs (hunk 1 above)
Comment (MartinBentley) [reply src/ManageCourses.Api/Services/Data/DataService.cs]: Done

File: tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs (hunk 3 above)
Comment (MartinBentley) [reply tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs]: Done

File: tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs (hunk 2 above)
Comment (MartinBentley) [reply tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs]: All done

**Pull request merged**